File: src/main/java/com/crafly/craflyserver/CraflyServerApplication.java
package com.crafly.craflyserver;

import org.springframework.beans.factory.annotation.Value;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.PropertySource;

@SpringBootApplication
public class CraflyServerApplication {

    public static void main(String[] args) {
        SpringApplication.run(CraflyServerApplication.class, args);
    }

}

File: src/main/kotlin/com/crafly/craflyserver/CraflyServerApplication.kt
package com.crafly.craflyserver;

import org.springframework.boot.runApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories

@SpringBootApplication
@EnableJpaRepositories
class CraflyServerApplication
fun main(args: Array<String>) {
    runApplication<CraflyServerApplication>(*args)
}
